src/software/jetson_nano/thunderloop.cpp

```diff
@@ -151,7 +195,7 @@ void Thunderloop::runLoop()
         // Make sure the iteration can fit inside the period of the loop
         CHECK(loop_duration * static_cast<int>(SECONDS_PER_NANOSECOND) <=
               (1.0 / loop_hz_))
-            << "Loop takes longer than 1/loop_hz_ seconds";
+            << "Thunderloop iteration took longer than 1/loop_hz_ seconds";
```

Comment:
looks like something that should be reported back to thunderscope

Reply:
it will be, all g3logs are sent back that's what the network logging part of this PR is for